twindle-cli/utils/send-email.js

```diff
@@ -2,25 +2,30 @@ const nodemailer = require("nodemailer");
 const emailConfig = require("../nodemailer.config.json");
 
 async function sendMail({ subject, emailTo, attachments }) {
-	// create reusable transporter object using the default SMTP transport
-	const transporter = nodemailer.createTransport(emailConfig);
+  // create reusable transporter object using the default SMTP transport
+  const transporter = nodemailer.createTransport(emailConfig);
 
-	// send mail with defined transport object
-	const info = await transporter.sendMail({
-		from: emailConfig.auth.user, // sender address
-		to: emailTo, // list of receivers
-		subject, // Subject line
-		attachments: [
-			...attachments,
-			// {
-			// 	filename: "twindle.epub",
-			// 	content: Buffer.from(file, "base64"),
-			// 	contentType: "application/epub+zip",
-			// },
-		],
-	});
+  console.log(attachments);
 
-	console.devLog("Email sent: %s", info.messageId);
+  // send mail with defined transport object
+  const info = await transporter.sendMail({
+    from: emailConfig.auth.user, // sender address
+    to: emailTo, // list of receivers
+    subject, // Subject line
+    attachments: [
+			{
+				
+			},
```

Comment:
@PuruVJ why is this added

Reply:
Just some testing. Irrelevant. Will remove

Reply:
Cool